cmake/platform/linux/wayland.cmake

```diff
@@ -1,6 +1,21 @@
-set(PLATFORM_REQUIRED_DEPS OpenGl EGL Waylandpp LibDRM Xkbcommon)
```

Comment:
I think that It's ok to make some sane default assumptions, but people have to explicitly select wayland anyway, can as well ask them to select GL/GLES. This would be a new option then - should it be specific to Wayland like `WAYLAND_RENDER_SYSTEM` or more general?